src/Data/Container/Sized.agda

```diff
@@ -0,0 +1,27 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Sized containers, based on the work of Abbott and others
```

Comment:
Hmm I'm not sure about this module name... It's not really about sized containers per se, only the implementation of the fixpoints. Would this be better as `Data.Container.Fixpoints.Sized`?